drink/src/session.rs

```diff
@@ -395,3 +470,71 @@ impl<R: Runtime> Session<R> {
         self.sandbox.override_debug_handle(d);
     }
 }
+
+/// A struct representing the result of parsing a `.contract` bundle file.
```

Comment:
I'd put all the code below (bundle-related) into a separate submodule in a `session/` folder - this file is long enough already :)